crypto/merklesignature/persistentMerkleSignatureScheme_test.go

```diff
@@ -114,3 +117,19 @@ func createTestDB(a *require.Assertions) *db.Accessor {
 
 	return &store
 }
+
+func getStateProofTableSchemaVersions(db db.Accessor) (int, error) {
+	var version int
+	err := db.Atomic(func(ctx context.Context, tx *sql.Tx) (err error) {
+		row := tx.QueryRow("SELECT version FROM schema where tablename = ?", merkleSignatureTableSchemaName)
+
+		err = row.Scan(&version)
+		if err != nil {
```

Comment:
if err is `sql.ErrNoRows` than you'd better return 0, since it just means that the database was never upgraded.

Reply:
just because it better aligns with the database versioning : 
no version means version 0; any other version means a specific version.
In your case, returning version 0 here would fail in 
```golang
a.Equal(merkleSignatureSchemaVersion, version)
```